docs/guides/capsule-update.md

```diff
@@ -41,6 +41,21 @@ should have this feature enabled.
 * _Firmware capsule_<br>
   The firmware update capsule file itself.  Should have `.cap` file extension.
 
+## List of items preserved after performing an update
```

Comment:
Maybe something like "What's preserved by an update", because the reader can see it's the list.

Reply:
https://github.com/Dasharo/docs/pull/912/commits/80dcc0ef34936f01a8f08d51fbba747f9f6656d6